go/cmd/dolt/commands/diff.go

```diff
@@ -148,7 +145,7 @@ func (cmd DiffCmd) ArgParser() *argparser.ArgParser {
 	ap.SupportsString(FormatFlag, "r", "result output format", "How to format diff output. Valid values are tabular, sql, json. Defaults to tabular.")
 	ap.SupportsString(whereParam, "", "column", "filters columns based on values in the diff.  See {{.EmphasisLeft}}dolt diff --help{{.EmphasisRight}} for details.")
 	ap.SupportsInt(limitParam, "", "record_count", "limits to the first N diffs.")
-	ap.SupportsFlag(CachedFlag, "c", "Show only the unstaged data changes.")
+	ap.SupportsFlag(cli.CachedFlag, "c", "Show only the unstaged data changes.")
```

Comment:
Looks like we had this doc string backwards! In git, `--cached` is a synonym for `--staged`, so supplying `--cached` means to do a diff using the staged content (and not the unstaged working set content). `dolt diff` is already working correctly, we just need to fix this backwards doc string. 

```suggestion
	ap.SupportsFlag(cli.CachedFlag, "c", "Show only the staged data changes.")
```